client/src/views/client-area/StatisticsPage.vue

```diff
@@ -2,30 +2,214 @@
+          </ion-card>
+          <ion-card class="card card-revoked">
+            {{ stats.standardUsersDetail.revoked }}
+            {{ $msTranslate(stats.standardUsersDetail.revoked > 1 ? 'clientArea.statistics.standards' : 'clientArea.statistics.standard') }}
+          </ion-card>
+          <ion-card class="card card-revoked">
+            {{ stats.outsiderUsersDetail.revoked }}
+            {{ $msTranslate(stats.outsiderUsersDetail.revoked > 1 ? 'clientArea.statistics.outsiders' : 'clientArea.statistics.outsider') }}
+          </ion-card>
+        </div>
+      </div>
+      <div class="bottom-part">
+        <h2>{{ $msTranslate('clientArea.statistics.titles.storage') }}</h2>
+        <div class="bottom-part-data">
+          <div>
+            <h2>{{ $msTranslate(formatFileSize(totalData)) }}</h2>
+            {{ $msTranslate('clientArea.statistics.ofWhich') }}
+            {{ $msTranslate(formatFileSize(stats.dataSize)) }} {{ $msTranslate('clientArea.statistics.data') }}
+            {{ $msTranslate(formatFileSize(stats.metadataSize)) }} {{ $msTranslate('clientArea.statistics.metadata') }}
+          </div>
+          <div>
+            {{ $msTranslate('clientArea.statistics.consumptionDetail') }}
+            <div>
+              <div id="firstBar">
+                <ion-progress-bar :value="firstBarData!.progress" />
+                {{ $msTranslate(formatFileSize(firstBarData!.amount)) }}
+                {{ firstBarData?.percentage.toFixed(0) }}%
+              </div>
+              {{ $msTranslate('clientArea.statistics.free') }}
+            </div>
+            <div v-if="secondBarData">
+              <div id="secondBar">
+                {{ $msTranslate(formatFileSize(secondBarData.amount)) }}
+                {{ secondBarData.multiplier! > 0 ? `x${secondBarData.multiplier}` : '' }}
+                {{ secondBarData.percentage.toFixed(0) }}%
+                <ion-progress-bar :value="secondBarData.progress" />
+              </div>
+              <div
+                v-if="thirdBarData"
+                id="thirdBar"
+              >
+                {{ $msTranslate(formatFileSize(thirdBarData.amount)) }}
+                {{ thirdBarData.percentage.toFixed(0) }}%
+                <ion-progress-bar :value="thirdBarData.progress" />
+              </div>
+              {{ $msTranslate('clientArea.statistics.paying') }}
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
 import { BmsAccessInstance, DataType, BmsOrganization, OrganizationStatsResultData } from '@/services/bms';
 import { onMounted, ref } from 'vue';
+import { IonCard, IonProgressBar, IonText } from '@ionic/vue';
+import { formatFileSize } from '@/common/file';
+
+interface ProgressBarData {
+  amount: number;
+  percentage: number;
+  progress: number;
+  multiplier?: number;
```

Comment:
Set `multiplier` to `0` instead of optional, so you can just do so you don't need the `!` when trying to display it.